certified-connectors/personr/personr.swagger.json

```diff
@@ -0,0 +1,487 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Personr API",
+    "description": "Create, verify and retrieve applicant data using Personr. Effortlessly comply with KYC/AML, using PowerAutomate to integrate into your own technology."
+  },
```

Comment:
Please add contact information, as per the template
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

Additionally, please refer the swagger file schema and update duly. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json